Jenkinsfile.ocp4

```diff
@@ -1319,18 +1319,18 @@ pipeline {
                         //     "--overwrite"
                         // )
 
-                        def docBackupCronJob = openshift.process("-f",
-                            "${templateDir}/jobs/minio-backup/minio-backup.cj.yaml",
-                            "NAME_SUFFIX=${prodSuffix}",
-                            "NAMESPACE=${prodProject}",
-                            "VERSION=v1.0.0",
-                            "SCHEDULE=15 4 * * *",
-                            "DEST_PVC=${nfsProdBackupPVC}",
-                            "SOURCE_PVC=gwells-minio",
-                            "PVC_SIZE=40Gi"
-                        )
+                        // def docBackupCronJob = openshift.process("-f",
```

Comment:
Do you want to leave these comments here?

Reply:
Just temporarily disabled, I have been uncommenting them ad-hoc within Jenkins to test them one at a time. I will uncomment once they're confirmed working.